### PR TITLE
Add gateway account onboarding events

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

**Review by viceromag — DISMISSED** [review]

**Review by iamdevice — DISMISSED** [review]

File: openapi/components/schemas/EventType.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ enum:
   - customer-one-time-password-requested
   - dispute-created
   - experian-check-performed
+  - gateway-account-activated
```
Comment (ashkarpetin) [inline openapi/components/schemas/EventType.yaml:12]: Is this event emitted every time gatewa account status is changed to active?

Comment (XaosSintez) [reply openapi/components/schemas/EventType.yaml:12]: We also have `gateway-account-enabled` event to handle situation when gateway account was disabled and enabled again.

So, `gateway-account-activated` event is emitted only for gateways with complex onboarding

Comment (ashkarpetin) [reply openapi/components/schemas/EventType.yaml:12]: I think the event name is confusing, perhaps `gateway-account-onboarding-completed` might be better.

Comment (XaosSintez) [reply openapi/components/schemas/EventType.yaml]: Applied `gateway-account-onboarding-completed`

**Review by ashkarpetin — APPROVED** [review]

**Review by wmdmgv — APPROVED** [review]

**Review by iamdevice — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Pull request merged**